praas-app-ionic/src/styles/common.scss

```diff
@@ -1,12 +1,24 @@
+html,
+body {
+  background-color: var(--ion-color-light);
+}
+
+ion-content {
+  --background: var(--ion-color-light);
+}
+
 .ion-page-invisible {
   // Quick fix: Ionic is not removing this class when navigating from one route to another
   opacity: 1 !important;
 }
-.text-align-center {
-  text-align: center;
-}
+
 .error {
   color: var(--ion-color-danger);
   font-size: 12px;
   text-align: left;
 }
+
+/* Utility Classes */
+.u-text-align-center {
```

Comment:
can we have a better prefix? something like we have for `ion`?

Reply:
It is better not to mess up with the ion namespace. Was following the .u style for utility .o style for object and .c style for component

Reply:
yeah, i din't mean to use `ion-` prefix. we should be having something like `praas-` than `.o`, `.u` which might end up messy in the end and difficult.